runtime/src/main/java/org/corfudb/runtime/clients/LayoutClient.java

```diff
@@ -27,13 +28,15 @@ public LayoutClient(IClientRouter router, long epoch, UUID clusterID) {
         super(router, epoch, clusterID);
     }
 
+    // Protobuf region
+
```

Comment:
Remove. All of this uses protobuf messages now. Also, don't forget to remove Layout requests/responses from CorfuMsgType.